src/omv/boards/OPENMV_RT1060/omv_boardconfig.h

```diff
@@ -211,4 +211,6 @@
 #define OMV_CSI_POWER_PIN               (&omv_pin_DCMI_POWER)
 #define OMV_CSI_FSYNC_PIN               (&omv_pin_DCMI_FSYNC)
 
+#define OMV_CSI_HAS_BSWAP               (1)
```

Comment:
```suggestion
#define OMV_CSI_HW_SWAP_ENABLE          (1)
```
Can you please add this right after `OMV_CSI_XCLK_FREQUENCY` ? Options first, then pins.

Reply:
Done

Reply:
Please use this option name `OMV_CSI_HW_SWAP_ENABLE`

Reply:
Done